### ISSUE TITLE
Missing sitecore_xxx_indexRebuildAlias

### ISSUE THREAD

**Issue by GoranHalvarsson** [LaubPlusCo/Docker-for-Sitecore-dev#1]
Mr Laub

Great job as always :-)

I had a small hickup, when I did an rebuild the Sitecore.ContentSearch.SolrCloud.SwitchOnRebuild.config tried to locate index, sitecore_(core/web/master)_indexRebuildAlias.

I had to add following indexes/cores:
sitecore_core_indexRebuildAlias, 
sitecore_master_indexRebuildAlias,
sitecore_web_indexRebuildAlias

Thanks to your readme it was very easy to add the new core's :-)

**Comment by anderslaub** [comment]
Mr Halvarsson, 

Good point and glad that you managed to add the cores easily.

I am not sure I will really consider it a bug though since switch on rebuild is not required in development which is why they were left out in the first place (I did not have switch on rebuild enabled on my dev setup). 

It doesnt hurt to have the cores defined though so I have added them and will commit them.

- Remember that you are also welcome to contribute :)
